### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by YadlaMani**
Fixed Typos

### CODE REVIEW

**Comment by origranot** [comment]
I think this can fit better:
```"Reduced.to is a modern web application that shortens the length of link URLs, making it easier to remember, share, and track."```

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@
 [![Preview](/assets/images/preview.gif)](https://reduced.to)
 
   <p align="center">
-    Reduced.to is a modern web application that reduces the length of link URL. So it's easier to remember, share and track.
+    Reduced.to is a modern web application that reduces the length of the link URL. So it's easier to remember, share and track.
```
Comment (origranot) [inline README.md:22]: I think this is a better explanation:

```suggestion
Reduced.to is a modern web application that shortens the length of link URLs, making it easier to remember, share, and track.
```

**Review by origranot — CHANGES_REQUESTED** [review]
@YadlaMani Thank you for your contribution, please take a look at my suggestion.

**Comment by YadlaMani** [comment]
@origranot do you want me to do this changes and commit

**Comment by origranot** [comment]
@YadlaMani Yea 💯

**Comment by origranot** [comment]
@YadlaMani Hey, 
Are you planing on push the changes?

**Comment by origranot** [comment]
> Yeah

Ok, I will wait for you to resolve the comment I left.

**Pull request closed without merging**